azkaban-common/src/main/java/azkaban/executor/ExecutionControllerUtils.java

```diff
@@ -324,7 +324,7 @@ public static void failEverything(final ExecutableFlow exFlow,
    */
   public static void failEverything(final EventHandler eventHandler,
       final ProjectManager projectManager, final ExecutableFlow exFlow,
-      final Status finalFlowStatus) {
+      final Status finalFlowStatus, final ExecutorLoader executorLoader) {
```

Comment:
If we are passing the `executorLoader` as a parameter perhaps we could also include the executable flow update call (`updateExecutableFlow(executableFlow)`) inside this method. What do you think about that refactor?